meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
 # Explicit machine declaration by build job:
 # MACHINE = "$TARGET_MACHINE"
 
+# Build number shared by all jobs across the different machines:
+# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
+
```

Comment:
On Thu, 2016-03-31 at 08:16 -0700, Alexander D. Kanevskiy wrote:
> In meta-ostro/conf/distro/include/ostroproject-ci.inc:
> 
> > @@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
> >  # Explicit machine declaration by build job:
> >  # MACHINE = "$TARGET_MACHINE"
> >  
> > +# Build number shared by all jobs across the different machines:
> > +# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
> > +
> 
> it is not explicitly set variable, it's exported from jenkins.

How is that different from, say, BB_NUMBER_PARSE_THREADS?

My understanding was that the "Dymamic section." contains "Values are
applied dynamically based on runtime config or builder host parameters."
and to me, that includes BUILD_ID.

I don't care who sets it, as long as it is documented and can be relied
upon to be in a certain format.

But sure, I can remove the entry and just document that OS_VERSION
depends on it.